sql-statements/sql-statement-split-region.md

```diff
@@ -375,11 +392,13 @@ You can specify the partition to be split. See the following usage example:
 
 ## pre_split_regions
 
-To have evenly split Regions when a table is created, it is recommended you use `shard_row_id_bits` together with `pre_split_regions`. When a table is created successfully, `pre_split_regions` pre-spilts tables into the number of Regions as specified by `2^(pre_split_regions)`.
+To have evenly split Regions when a table is created, it is recommended you use `SHARD_ROW_ID_BITS` together with `PRE_SPLIT_REGIONS`. When a table is created successfully, `PRE_SPLIT_REGIONS` pre-spilts tables into the number of Regions as specified by `2^(PRE_SPLIT_REGIONS)`.
 
 > **Note:**
 >
-> The value of `pre_split_regions` must be less than or equal to that of `shard_row_id_bits`.
+> The value of `PRE_SPLIT_REGIONS` must be less than or equal to that of `SHARD_ROW_ID_BITS`.
+
+The `tidb_scatter_region` global variable affects the behavior of `PRE_SPLIT_REGIONS`. This variable controls the time of returning the results after the table is created. If there are a large number of writes after creating the table, you need to set the value of this variable to `1`, so TiDB does not return the results to the client until all the regions have been split and scattered. Otherwise, TiDB writes the data before the scattering is completed, which will have a significant impact on write performance.
```

Comment:
```suggestion
The `tidb_scatter_region` global variable affects the behavior of `PRE_SPLIT_REGIONS`. This variable controls whether to wait for Regions to be pre-split and scattered before returning results after the table creation. If there are intensive writes after creating the table, you need to set the value of this variable to `1`, then TiDB will not return the results to the client until all the Regions are split and scattered. Otherwise, TiDB writes the data before the scattering is completed, which will have a significant impact on write performance.
```